lib/rack/profiler.rb

```diff
@@ -36,9 +37,10 @@ def initialize(app, &block)
     def call(env)
       @events = []
       req = Rack::Request.new(env)
+      env['rack-profiler'] = self
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.